src/main/include/wrapperclasses/Motor.h

```diff
@@ -0,0 +1,57 @@
+#pragma once 
+
+class Motor {
+    public:
+        Motor(); 
+
+        virtual void SetPercent(double percent) = 0;
+        virtual void StopMotor() = 0; 
+
+        virtual void Configure() = 0; 
+
+        // ENCODER FUNCTIONS // 
+
+        virtual double GetRelativeVelocity() = 0; 
+        virtual double GetRelativePosition() = 0; 
+
+        virtual double SetRelativePosition(double pos) = 0; 
+
+        // END ENCODER FUNCTIONS // 
```

Comment:
Probably want to include absolute encoder and hardware limit switch getters as well.